scripts/release-build.js

```diff
@@ -38,7 +43,11 @@ async function copyFiles() {
 
 function createManifest() {
   const full = require('../package.json');
-  const minimal = { main: 'index.js' };
+  const minimal = {
+    main: 'index.cjs',
+    exports: { import: './index.js', require: './index.cjs' },
```

Comment:
Just a note for @ddbeck -- with the inclusion of an `exports` in our `package.json`, we'll need to bump the major semver.  An [article](https://redfin.engineering/node-modules-at-war-why-commonjs-and-es-modules-cant-get-along-9617135eeca1#1152) I have been reading states the following:

> Beware: adding an exports map is always a “semver major” breaking change. By default, your users can reach into your package and require() any script they want, even files that you intended to be internal. The exports map ensures that users can only require/import the entry points that you deliberately expose.

While we don't really expose any other files besides `data.json` and `index.js`, it is a potentially breaking change.  But despite that, it feels fitting to bump the major version regardless -- ESM support is a big deal after all!